src/Neuron/Zettelkasten/View.hs

```diff
@@ -91,7 +92,10 @@ renderZettel (store, graph) zid = do
   div_ [class_ "zettel-view"] $ do
     div_ [class_ "ui raised segment"] $ do
       h1_ [class_ "header"] $ toHtml zettelTitle
-      MMark.render $ useExtension (linkActionExt store) zettelContent
+      MMark.render
+        $ useExtension (linkActionExt store)
+        $ useExtension (mathJax (Just '$'))
+        $ zettelContent
```

Comment:
Hey, thanks for the PR! Since neuron uses `rib`, and as `rib` already includes mathJax in [its list of default extensions](https://github.com/srid/rib/blob/48c0e70/src/Rib/Parser/MMark.hs#L119), this part of the change is not necessary. Can you confirm that it works without it (and the cabal file change)? 

If yes, then we only need to merge the `script_` include of mathjax Javascript. I'm not against including it; but I might disable its inclusion _by default_ unless explicitly configured by the user (see #15).

Reply:
You are right, the script include is sufficient. An option to enable it explicitly would be nice though